src/resources/books.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: default
 permalink: /resources/books
-title: Books about Dart
+title: Books and Videos about Dart
 description: Read all about it! Here's a collection of books about Dart.
 toc: false
 ---
 
-Here's a collection of books about Dart.
+Here's a collection of books and videos about Dart.
```

Comment:
I thought about this, too. We don't want to include  youtube videos or Dart Tips, for example. It seems like we shouldn't have to explain it but I could, perhaps, add "packaged" videos. Not loving it, though. I could say, "professionally produced books and videos", but I'm not loving that, either. I think we should leave it be and just vet any thing that comes in.